src/stacks-hierarchy/Root_PurchaseOverviewScreen/Root_PurchaseOverviewScreen.tsx

```diff
@@ -49,7 +49,11 @@ export const Root_PurchaseOverviewScreen: React.FC<Props> = ({
     ? 'isFree' in params.toPlace && !!params.toPlace.isFree
     : false;
 
-  const {selection, preassignedFareProductAlternatives} = useOfferDefaults(
+  const {
+    selection,
+    preassignedFareProductAlternatives,
+    selectableUserProfiles,
+  } = useOfferDefaults(
```

Comment:
Rather do this outside of the `useOfferDefaults`. The `useOfferDefaults` will be deleted soon when continuing the refactor utilising the `PurchaseSelectionType`.